easybuild/easyconfigs/m/MATLAB-Engine/MATLAB-Engine-2021a-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'MATLAB-Engine'
+version = '2021a'
+
+homepage = 'https://www.mathworks.com/help/matlab/matlab-engine-for-python.html'
+description = """The MATLAB Engine API for Python provides a package for Python
+ to call MATLAB as a computational engine."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+dependencies = [
+    ('MATLAB', version, '', SYSTEM),
+    ('Python', '3.8.6'),
+]
+
+use_pip = False
+download_dep_fail = True
+
+# Since this setup.py does not provide a separate --build-base for the install
+# step, both build and install must be performed in a single command.
+prebuildopts = "cd $EBROOTMATLAB/extern/engines/python && "
+buildopts = "--build-base=%(builddir)s install --prefix=%(installdir)s"
```

Comment:
It would be better to copy the sources to `builddir` to be sure that nothing is modified in the existing installation of MATLAB. The following does the trick, what do you think?
```suggestion
# Copy sources from MATLAB root to build dir
prebuildopts = "cp -r $EBROOTMATLAB/extern/engines/python/* %(builddir)s/ && "
prebuildopts += 'sed -i "s|_cwd,.*_parent|\'$EBROOTMATLAB\'|" setup.py && '
# Since this setup.py does not provide a separate --build-base for the install
# step, both build and install must be performed in a single command.
buildopts = "--build-base=%(builddir)s install --prefix=%(installdir)s"
```

Reply:
wait, we've been doing this wrong... MATLAB-Engine is actually available from [PyPI](https://pypi.org/project/matlabengine) and Mathworks has a [github repo](https://github.com/mathworks/matlab-engine-for-python) with the source code.

Those packages do even receive updates and are compatible with `pip`. So there is no reason to use the sources in the installation of MATLAB.

I tested it and it works very well. I made a PR to this PR with the updated easyconfigs. Please check https://github.com/Micket/easybuild-easyconfigs/pull/13